test/global/run/uses_old_lockfile_test.dart

```diff
@@ -1,55 +0,0 @@
-// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
```

Comment:
We should still behave nicely with old-style packages—ideally recompile them, if not that then create a `.packages` file, or at the very least produce a useful error message telling users how to fix the problem.